tests/dart-2.16/lib/tests.dart

```diff
@@ -15,17 +15,17 @@ import 'package:dio/dio.dart' hide Response;
 */
 
 Future<void> start(final req, final res) async {
-
   final payload = jsonDecode(req.payload == '' ? '{}' : req.payload);
 
   final id = payload['id'] ?? '1';
-  final todo = await Dio().get('https://jsonplaceholder.typicode.com/todos/$id');
-
+  final todo =
+      await Dio().get('https://jsonplaceholder.typicode.com/todos/$id');
+  print('stdout');
```

Comment:
Can we add more prints to test? primitive types + object + array. If behavior is different from other runtimes, it might be fine, we will do a specific test for that runtime. The main idea is to fully understand what will be returned.